torch/_refs/__init__.py

```diff
@@ -4647,6 +4648,48 @@ def triu_indices(
     )
 
 
+@register_decomposition(torch.ops.aten.bucketize)
+@out_wrapper()
+def bucketize(
+    a: TensorLikeType,
+    boundaries: TensorLikeType,
+    *,
+    out_int32: bool = False,
+    right: bool = False,
+):
```

Comment:
Should this check that boundaries is a one-dimensional tensor? 

Let's also comment that boundaries is assumed to be an increasing sequence. The current documentation for bucketize says the sequence must be "monotically" increasingly, which I think is more typically described as "weakly" increasing -- but is this true? We should be clear here. Also, what happens if a non-increasing 1D tensor is given? Are the values in the result tensor undefined?

Reply:
Right; I don't want to validate that the tensor's data is what we expect, just let the users know what will happen if it's not.

Reply:
Ok, I added a comment to the docs about how result is undefined if boundaries is not increasing.